README.md

```diff
@@ -12,13 +14,40 @@ npm install
 bower install
 ```
 
+The backend is supported by Docker.
+
+To install Docker:
+  * For Linux, go [here](https://docs.docker.com/engine/installation/). 
+  * For OS X, go [here](https://docs.docker.com/engine/installation/mac/).
+  * For Windows, go [here](https://docs.docker.com/engine/installation/windows/).
+
+You might also want to install Devtools for [React](https://facebook.github.io/react/blog/2015/09/02/new-react-developer-tools.html). For Redux, you can install the [Google Chrome extension](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd) or use the other methods described in the README [here](https://github.com/zalmoxisus/redux-devtools-extension).
+
+####To run the Docker server
+
+Go to the project directory and follow the instructions below:
+
+1. Open up the Docker Quickstart Terminal and wait for the machine to start running.
```

Comment:
all of this is outdated. the new containers can be init'ed just by running `docker-compose up` and then `./init_docker.sh` *(as long as you're using the docker for mac beta)